app/main.ipynb

```diff
@@ -493,18 +705,50 @@
     "    result = total / len(lst)\n",
     "    return result\n",
     "\n",
-    "lst = [1, 2, 3, 4, 5]"
+    "lst = [1, 2, 3, 4, 5]\n",
+    "\n",
+    "result = unknown_signature(lst)\n",
+    "\n",
+    "%time unknown_signature(lst)\n",
+    "result"
    ],
    "metadata": {
     "collapsed": false
    }
   },
   {
    "cell_type": "code",
-   "execution_count": 28,
-   "outputs": [],
+   "execution_count": 125,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 24 µs, sys: 1 µs, total: 25 µs\n",
+      "Wall time: 26.7 µs\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": "3.0"
+     },
+     "execution_count": 125,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your numpy code here and estimate the time of execution"
+    "def unknown_signature(lst):\n",
+    "    lst = np.array(lst)\n",
+    "    result = np.mean(lst)\n",
+    "    return result\n",
+    "\n",
+    "lst = [1, 2, 3, 4, 5]\n",
```

Comment:
The same comment as above

Reply:
I need ur explanation because the function transforms the list in an array